control/controlclient/direct.go

```diff
@@ -1151,8 +1151,8 @@ var clockNow = time.Now
 
 // opt.Bool configs from control.
```

Comment:
it does say "opt.Bool" here.

The comment is _factored_ out! 😂

Reply:
too far, doesn't follow standards all over the code base :P